packages/wonder-blocks-icon-button/src/components/node-icon-button.tsx

```diff
@@ -0,0 +1,316 @@
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import {Link} from "react-router-dom-v5-compat";
+
+import {StyleType, View} from "@khanacademy/wonder-blocks-core";
+import {PhosphorIcon} from "@khanacademy/wonder-blocks-icon";
+import {focusStyles} from "@khanacademy/wonder-blocks-styles";
+import {border, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+
+import type {BaseIconButtonProps} from "../util/icon-button.types";
+
+import {IconButtonUnstyled} from "./icon-button-unstyled";
+
+/**
+ * The object containing the CSS variables that can be overridden to customize
+ * the appearance of the NodeIconButton component.
+ */
+type Tokens = Partial<{
+    "--wb-node-icon-button--box-foreground": string;
+    "--wb-node-icon-button--box-background": string;
+    "--wb-node-icon-button--box-shadow-color": string;
+    "--wb-node-icon-button--box-padding": string | number;
+    "--wb-node-icon-button--box-shadow-y-rest": string | number;
+    "--wb-node-icon-button--box-shadow-y-hover": string | number;
+    "--wb-node-icon-button--box-shadow-y-press": string | number;
+    "--wb-node-icon-button--icon-size": string | number;
+}>;
+
+type Props = Omit<BaseIconButtonProps, "kind" | "style"> & {
+    /**
+     * The action type of the button. This determines the visual style of
+     * the button. Defaults to `notStarted`.
+     *
+     * - `notStarted` is used for buttons that indicate a not started action.
+     * - `attempted` is used for buttons that indicate an attempted (in progress)
+     *   action.
+     * - `complete` is used for buttons that indicate a complete action.
+     */
+    actionType?: "notStarted" | "attempted" | "complete";
+    /**
+     * The alternative text for the icon button. Required for accessibility.
+     */
+    "aria-label": string;
+
+    /**
+     * The size of the icon button.
+     * One of `small` (24) or `large` (68).
```

Comment:
The `small` size is smaller than the [44x44 px min target size for a11y](https://www.w3.org/TR/WCAG21/#target-size). Should we create a target area around it for touch interactions? Or maybe the button size needs to be increased?